doc/source/conf.py

```diff
@@ -98,6 +98,19 @@
 # copybutton config: strip console characters
 copybutton_prompt_text = r">>> |\.\.\. |\$ |In \[\d*\]: | {2,5}\.\.\.: | {5,8}: "
 copybutton_prompt_is_regexp = True
+# numpydoc configuration
+numpydoc_xref_param_type = True
+numpydoc_xref_ignore = {
+    "of", "or", "optional", "default", "1D", "2D", "3D", "n-dimensional", "M", "N", "K",
+}
+numpydoc_xref_aliases = {
+    "Labeller": ":ref:`Labeller <labeller_api>`",
+    "ndarray": ":class:`~numpy.ndarray`",
+    "InferenceData": ":class:`~arviz.InferenceData`",
+    "matplotlib_axes": ":class:`matplotlib Axes <matplotlib.axes.Axes>`",
+    "bokeh_figure" ":class:`Bokeh Figure <bokeh.plotting.Figure>`",
```

Comment:
```suggestion
    "bokeh_figure": ":class:`Bokeh Figure <bokeh.plotting.Figure>`",
```

There is a colon missing which breaks the parsing of the code.